samples/bluetooth/unicast_audio_server/src/main.c

```diff
@@ -29,8 +29,12 @@
 				  BT_AUDIO_CONTEXT_TYPE_MEDIA | \
 				  BT_AUDIO_CONTEXT_TYPE_GAME)
 
-/* Mandatory support preset by both client and server */
-static struct bt_audio_lc3_preset preset_16_2_1 = BT_AUDIO_LC3_UNICAST_PRESET_16_2_1;
+#define CHANNEL_COUNT_1 BIT(0)
```

Comment:
Ideally we'll provide helper functions to help format things like this, but this is fine for now